pkg/controller/kfservice/kfservice_controller.go

```diff
@@ -99,6 +100,11 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// TODO enqueue all kfservices to reconcile
```

Comment:
going to do in this PR or subsequent?

Reply:
Yes, I am bloating up the size of this PR a bit now.

Reply:
create an issue and tag it?

Reply:
https://github.com/kubeflow/kfserving/issues/139

Reply:
https://github.com/kubeflow/kfserving/issues/139